### PR TITLE
Update SECURITY.md

### PR DESCRIPTION
**Pull request by johnrballard**
Added github advisory reporting for security related findings.

### CODE REVIEW

**Review by workingjubilee — APPROVED** [review]
Looks good.

File: SECURITY.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 **Please do not report security vulnerabilities through public GitHub issues.**
 
-Instead, please send an email to secure@tcdi.com to report the security issue. You should receive a response within 3 working days. If for some reason you do not, please follow up via email to ensure we received your original message. 
+Instead, please open a [security advisory][advisory] which notify the maintainers. You should receive a response within 3 working days. If for some reason you do not, please follow up via email to ensure we received your original message.
```
Comment (eeeebbbbrrrr) [inline SECURITY.md:7]: "which **notifies**"?

**Review by eeeebbbbrrrr — CHANGES_REQUESTED** [review]

File: SECURITY.md (hunk 1 above)
Comment (workingjubilee) [reply SECURITY.md:7]: could also be "to notify"

Comment (johnrballard) [reply SECURITY.md:7]: Shoot.  Didn't catch that.

**Review by eeeebbbbrrrr — APPROVED** [review]

**Pull request merged**